core/quill.js

```diff
@@ -64,6 +65,7 @@ class Quill {
   }
 
   constructor(container, options = {}) {
+    this.formattingStarted = false;
```

Comment:
It looks like some debug row forgotten to be deleted :)

Reply:
yes. it's from prev changes. will fixed